file_paths.md

```diff
@@ -163,156 +163,167 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
-            routes.yaml                       # :route_file
-            ssl                               # :ssldir
-    -------------------------------------------------------------------------------------------------------
-        var *
-            log                               # :logdir                      same
-                puppet.log                    # not enabled by default
-            run                               # :rundir                      same
-                agent.pid                     # :pidfile
-
-    C:\ProgramData\PuppetLabs\pxp-agent *
-        etc *
-            pxp-agent.conf                    # pxp-agent configuration file
-            modules *                         # stores configuration files for pxp-agent modules
-                pxp-module-puppet.conf        # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
-        var *
-            log *
-                pxp-agent.log                 # enabled by default
-            spool *                           # directory containing results of pxp-agent modules
-            run *
-
-    C:\Program Files\Puppet Labs\Puppet
-        VERSION                               # puppet-agent package version
-
-    C:\Program Files\Puppet Labs\Puppet\bin *
-        environment.bat *                     # setup LOAD_PATH
-        facter.bat *                          # bat file wrappers
-        hiera.bat *
-        mco.bat *
-        puppet.bat *
-        puppet_shell.bat *                    # targets for shortcuts
-        run_facter_interactive.bat *
-        run_puppet_interactive.bat *
-
-    C:\Program Files\Puppet Labs\Puppet\facter *
-        bin *                                 # executables and dlls
-            facter.exe *
-            libfacter.so *
-            lib*.dll *                        # gcc, mingw, boost libraries
-        inc *                                 # facter headers
-            facter *
-        lib *
-            facter.rb *                       # ruby bindings
 
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
-
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
-
-    C:\Program Files\Puppet Labs\Puppet\misc *
-        LICENSE.rtf *                         # license
-        puppetlabs.ico *                      # icon for start menu shortcut
-        puppetres.dll *                       # event log message resource dll
-        versions.txt *                        # versions of components
+    C:\Program Files\Puppet Labs\puppet *     # ruby-puppet root
+        bin *                                 # executables and dlls
+          c_rehash *
+          catstomp *
+          catstomp.bat *
+          curl.exe *
```

Comment:
purposefully... https://github.com/puppetlabs/puppet-agent/blob/feature/master/windows-vanagon-development/configs/components/curl.rb the curl executable is also available in *nix puppet-agent packages